src/components/calcite-inline-editable/calcite-inline-editable.scss

```diff
@@ -1,53 +1,56 @@
 :host([scale="s"]) {
-  .calcite-inline-editable__controls-wrapper {
+  .controls-wrapper {
     @apply h-6;
   }
 }
 
 :host([scale="m"]) {
-  .calcite-inline-editable__controls-wrapper {
+  .controls-wrapper {
     @apply h-8;
   }
 }
 
 :host([scale="l"]) {
-  .calcite-inline-editable__controls-wrapper {
+  .controls-wrapper {
     @apply h-11;
   }
 }
 
 :host(:not([editing-enabled]):not([disabled])) {
-  .calcite-inline-editable__wrapper {
+  .wrapper {
     &:hover {
       @apply bg-foreground-2;
     }
   }
 }
 
-.calcite-inline-editable__wrapper {
+.wrapper {
   @apply box-border
     flex
     justify-between
     bg-foreground-1
     transition-default;
 
-  .calcite-inline-editable__input-wrapper {
+  .input-wrapper {
     @apply flex-1;
   }
 }
 
-.calcite-inline-editable__controls-wrapper {
+.controls-wrapper {
   @apply flex;
 }
 
 :host([disabled]) {
-  .calcite-inline-editable__cancel-editing-button-wrapper {
+  .cancel-editing-button-wrapper {
     @apply border-color-2;
   }
 }
 
 :host {
   &::slotted(*) {
+    // todo: major refactoring of styling.
+    // Looks like inline-editable is modifying internals of input
+    // We need to have another way to do this instead
     .calcite-input__element-wrapper {
       textarea,
```

Comment:
@eriklharper did you work on the `inline-editable` or was that someone else?

We will need the styles refactored because they are currently styling internals of the input component. Since we are shadowing these, we can't do that anymore.

Can someone work on refactoring these styles?

Reply:
@alexabreu originally wrote this component, so I'll defer to him on that.

Reply:
> Can someone work on refactoring these styles?

I'd be happy to contribute to this effort if that would be helpful.

Reply:
@caripizza that would be awesome.

The issue is that the current inline-editable is modifying the internals of the input.

Reply:
@driskull should I push to a new branch based off this one, or what type of work flow are you thinking would work best for any style fixes?

Reply:
@caripizza can you push to this one please.

I'm thinking ideally that the 'calcite-input' isn't modified using any styles. If we need to pass props to it that would be fine.